src/app/dataValidator/index.js

```diff
@@ -9,8 +10,43 @@ const validateData = data => {
   log('\nValidation complete!');
 };
 
-const data = require('../../../data/scenario-01.json');
+const validateFile = filename => {
+  const data = fs.readFileSync(filename);
+  validateData(JSON.parse(data));
+};
+
+const readAllFiles = (filenames, dirname) =>
+  filenames.forEach(filename => {
+    // explicitly ignore scenario-23
+    if (filename.includes('scenario-23.json')) {
+      return;
+    }
+
+    if (filename.includes('.json')) {
+      validateFile(`${dirname}/${filename}`);
+    }
+  });
 
-validateData(data);
+const readdirAsync = dirname =>
+  new Promise((resolve, rejects) => {
+    fs.readdir(dirname, (err, filenames) => {
+      if (err) {
+        rejects(err);
+      } else {
+        resolve(readAllFiles(filenames, dirname));
+      }
+    });
+  });
 
-module.exports.validateData = validateData;
+console.time('readdirAsync'); // eslint-disable-line no-console
+readdirAsync('./././data').then(() => {
```

Comment:
Probably out of scope of this PR, but the script is very loud now that we test all files, we should put verbose logging behind a debugging flag.

Reply:
This will be handled as part of https://github.com/bbc/simorgh/issues/239